src/modules/Main.ui.js

```diff
@@ -68,8 +70,8 @@ import WalletName from './UI/components/Header/WalletName/WalletNameConnector.js
 import HelpModal from './UI/components/HelpModal'
 import { passwordReminderModalConnector as PasswordReminderModal } from './UI/components/PasswordReminderModal/indexPasswordReminderModal.js'
 import TransactionAlert from './UI/components/TransactionAlert/TransactionAlertConnector'
-import type { Permission } from './UI/permissions.js'
 import { CAMERA, CONTACTS } from './UI/permissions.js'
+import type { Permission } from './UI/permissions.js'
```

Comment:
`import { CAMERA, CONTACTS, type Permission } from './UI/permissions.js'` works since the import-sort upgrade.